Changes for vite-sentry plugin (#660)
epicweb-dev/epic-stack: docs/monitoring.md

To generate the auth token, click
[this](https://sentry.io/orgredirect/settings/:orgslug/developer-settings/new-internal/) to create an internal integration (which grants the selected capabilities to the recipient, similar to how RBAC works). Give it a name and add the scope for
`Releases:Admin`. Press Save, find the auth token at the bottom of the page
under "Tokens", and copy that to secure location (this becomes
`SENTRY_AUTH_TOKEN`). Then visit the organization settings page and copy that
organization slug (`SENTRY_ORG`), and the slug name for your project too (`SENTRY_PROJECT`).

Uncomment the sentry-vite plugin in `vite.config` and the build command in the 'build' section of the Dockerfile, which shows how to pass the `SENTRY_AUTH_TOKEN` (below) as a docker secret, so it is available to the vite config when `npm run build` is run (you will need to do the same for `SENTRY_ORG` and `SENTRY_PROJECT`). Note that these do not need to be added to the `env.server` env vars schema, as they are only used during the build and not the runtime.

> The plugin will create sentry releases for you and automatically associate commits during the vite build if you set up a GitHub integration with it. In this setup we have utilised a simple strategy for naming releases of using the commit sha, passed in as a build arg via the GitHub action workflow.

Reviewer: ```suggestion
The plugin will create sentry releases for you and automatically associate commits during the vite build once you set the `SENTRY_AUTH_TOKEN` in [your GitHub action secrets](https://docs.github.com/en/actions/security-guides/using-secrets-in-github-actions). In this setup we have utilized a simple strategy for naming releases of using the commit sha, passed in as a build arg via the GitHub action workflow.
```
